server/updateReferralStatus/updateReferralStatusView.ts

```diff
@@ -0,0 +1,89 @@
+import UpdateReferralStatusPresenter from './updateReferralStatusPresenter'
+import ViewUtils from '../utils/viewUtils'
+
+export default class UpdateReferralStatusView {
+  constructor(private readonly presenter: UpdateReferralStatusPresenter) {}
+
+  getCurrentStatusTimelineOptions(htmlContent: string) {
```

Comment:
Like the comment below, can also enforce the type here in the future, but would need to be a custom type possibly